config/src/main/java/org/hyperledger/besu/config/JsonUtil.java

```diff
@@ -50,13 +50,28 @@ public static ObjectNode normalizeKeys(final ObjectNode objectNode) {
               final String normalizedKey = key.toLowerCase(Locale.US);
               if (value instanceof ObjectNode) {
                 normalized.set(normalizedKey, normalizeKeys((ObjectNode) value));
+              } else if (value instanceof ArrayNode) {
+                normalized.set(normalizedKey, normalizeKeysInArray((ArrayNode) value));
               } else {
                 normalized.set(normalizedKey, value);
               }
             });
     return normalized;
   }
 
+  private static ArrayNode normalizeKeysInArray(final ArrayNode arrayNode) {
+    final ArrayNode normalizedArray = JsonUtil.createEmptyArrayNode();
+    arrayNode.forEach(
+        value -> {
+          if (value instanceof ObjectNode) {
+            normalizedArray.add(normalizeKeys((ObjectNode) value));
+          } else if (value instanceof ArrayNode) {
+            normalizedArray.add(normalizeKeysInArray((ArrayNode) value));
+          }
```

Comment:
I think this might be missing an else case? 
e.g. if you had a string value in an array, rather than an Object or Array, it would get ignored and so effectively stripped out.

e.g.
```
"configArray": [
  "I don't need normalising but I shouldn't be stripped out"
]
```

yields
```
"configarray": []
```

Reply:
Nice catch!